### PR TITLE
Changing the mpn value to ean and referenceId as fallback

### PR DESCRIPTION
**Pull request by Milleo**
Version: 0.1.2
Reason: We need this update as requested internally for our team
Release: Minor

### CODE REVIEW

**Comment by Milleo** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [X]  Patch
> * [ ]  Minor
> * [ ]  Major
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by Milleo** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [X]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2020-03-27
```
Comment (klzns) [inline CHANGELOG.md]: There's no need to add this. Our bot add this automatically.
```suggestion
```

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "flixmedia",
   "vendor": "vtex",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (klzns) [inline manifest.json]: Same thing. Our bot change this.
```suggestion
```

**Review by klzns — APPROVED** [review]

**Pull request merged**